packages/abstractions/src/apiClientProxifier.ts

```diff
@@ -204,6 +204,10 @@ export const apiClientProxifier = <T extends object>(requestAdapter: RequestAdap
 						return apiClientProxifier(requestAdapter, downWardPathParameters, navigationCandidate.navigationMetadata, navigationCandidate.requestsMetadata);
 					};
 				}
+        if (name === "then") {
```

Comment:
prettier will most likely complain about the indent. You can fix it by running `npm run prettier:write`